spyder/plugins/editor/lsp/client.py

```diff
@@ -239,6 +241,8 @@ def on_msg_received(self):
                 if 'error' in resp:
                     logger.debug('{} Response error: {}'
                                  .format(self.language, repr(resp['error'])))
+                    if self.language == 'python':
+                        self.sig_lsp_error.emit(repr(resp['error']))
 
                 if 'method' in resp:
```

Comment:
I think we should change this to

    elif 'method' in resp:

because that's another possible response besides `error`.

@andfoy, is this correct?

Reply:
Pinging @andfoy again about this.

Reply:
Then we should send the signal inside the next if block (`if 'method' in resp:`) ?

Reply:
No, the signal is fine where it is. The thing is I really don't need to have:

```python
if 'error' in resp:
   ...
if 'method' in resp:
   ...
```

Instead, I think it should be

```python
if 'error' in resp:
   ...
elif 'method' in resp:
   ...
```

Reply:
I think that we are able to make that change

Reply:
@steff456 is completely right

Reply:
Great, thanks!